files/50unattended-upgrades

```diff
@@ -1,4 +1,3 @@
-/etc/apt/apt.conf.d/50unattended-upgrades 
 // Automatically upgrade packages from these (origin:archive) pairs
 Unattended-Upgrade::Allowed-Origins {
 	"${distro_id}:${distro_codename}-security";
```

Comment:
@sirv1ne what's the rationale? Do developers have another route to update `.deb`s?